api/src/test/java/org/apache/cloudstack/api/command/user/network/CreateNetworkCmdTest.java

```diff
@@ -100,12 +100,20 @@ public void testGetNetworkName() {
         Assert.assertEquals(cmd.getNetworkName(), netName);
     }
 
-    public void testGetDisplayText() {
+    public void testGetDisplayTextWhenNotEmpty() {
         String description = "Isolated Network";
         ReflectionTestUtils.setField(cmd, "displayText", description);
         Assert.assertEquals(cmd.getDisplayText(), description);
     }
 
+    public void testGetDisplayTextWhenEmpty() {
+        String description = null;
+        String netName = "net-isolated";
+        ReflectionTestUtils.setField(cmd, "displayText", description);
+        ReflectionTestUtils.setField(cmd, "name", netName);
+        Assert.assertEquals(cmd.getDisplayText(), netName);
+    }
```

Comment:
i think this is not the behaviour we want to test. Instead I think we need to
```suggestion
    public void testGetDisplayTextWhenEmpty() {
        String description = null;
        String netName = "net-isolated";
        ReflectionTestUtils.setField(cmd, "name", netName);
        Assert.assertEquals(cmd.getDisplayText(), netName);
    }
```